docs/content/guides/observability/ui/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Gloo UI
+weight: 
+description: You can install the Gloo UI with a separate Helm chart to get an at-a-glance view of the configuration, health, and compliance status of your Gloo Gateway setup and the workloads in your cluster.
+---
+
+You can install the Gloo UI with a separate Helm chart to get an at-a-glance view of the configuration, health, and compliance status of your Gloo Gateway setup and the workloads in your cluster.
+
+![Gloo UI dashboard]({{< versioned_link_path fromRoot="/img/ui-dashboard.png" >}})
```

Comment:
maybe add the graphic after the links, because from the landing page, I didn't realize there was more content after the graphic 
![image](https://github.com/user-attachments/assets/dae92a23-3fd8-4593-af02-19ab5a59d68b)